micrometer-core/src/main/java/io/micrometer/core/instrument/logging/LoggingMeterRegistry.java

```diff
@@ -199,12 +202,12 @@ private String humanReadableByteCount(double bytes) {
             int unit = 1024;
             if (bytes < unit) return bytes + " B";
             int exp = (int) (Math.log(bytes) / Math.log(unit));
-            String pre = "KMGTPE" .charAt(exp - 1) + "i";
+            String pre = "KMGTPE".charAt(exp - 1) + "i";
             return String.format("%.2f %sB", bytes / Math.pow(unit, exp), pre);
         }
 
         private String humanReadableBaseUnit(double value) {
-            if (" bytes" .equals(baseUnit)) {
+            if (" bytes".equals(baseUnit)) {
```

Comment:
Looks like this space might be a bug.

Reply:
@jkschneider If you meant the space in the " bytes", it aligns with the current logic. I wanted to straighten it by removing the space but didn't touch just in case there's any preference. If you agree with the removal of the space, I can rework it in this polish PR. Just let me know.

Reply:
@jkschneider I removed the space in 8899e9ac97c9099db4b60eee0ab4c0b3ec87b92e.